Source/SpacePort/XUnitTest.Test/UnitTest1.cs

```diff
@@ -0,0 +1,96 @@
+using RestSharp;
+using SpacePort;
+using SpacePort.Models;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+
+namespace XUnitTest.Test
+{
+    public class UnitTest1
+    {
+        [Fact]
+        public void CreateSpacePort()
+        {
+            //Arrange
```

Comment:
Kanske lite väl enkla unittester vissa av dom

Reply:
Beror på att vi provar TDD från scratch, vilket innebär att man skapar/genererar metoder, klasser, variabler utifrån testet :)
Helt onödigt med tid enligt min mening, som vi upptäckte en bit in på projektet